client/package.json

```diff
@@ -3,25 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@google-cloud/firestore": "^4.0.0",
-    "@google-cloud/storage": "^5.1.1",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
-    "bootstrap": "^4.5.0",
-    "create-react-class": "^15.6.3",
-    "file-saver": "^2.0.2",
-    "firebase": "^7.15.5",
-    "firebase-admin": "^8.12.1",
-    "literallycanvas": "^0.5.2",
-    "query-string": "^6.13.1",
```

Comment:
not sure why these dependencies are all removed. Are they being added in somewhere else?

Reply:
i made a mistake when i was merging and the pom.xml got all messed up so i reverted to what i had before i merged and then remerged to fix it. i didn't look inside the client, i did git pull multiple times as well to make sure i wasn't using any old stuff. is that what's showing up in the new commit I have?

Reply:
Here it shows what will be pushed https://github.com/googleinterns/step119-2020/pull/7/files
and it removes all the client files

Reply:
it might behave like that because it treats this as a "resolved" merge conflict already

Reply:
okay so besides these files here, is there anything else missing from the client so that i can add them manually? my git fetch doesn't change anything when I run it and git pull says everything is up to date

Reply:
it's probably easier to first copy your pom xml into an outside doc, revert to before you reverted, and then copy paste in the fixed pom xml. so git doesnt need to track commits of undoing and redoing, and it's only 1 file to copy paste rather than like 20